cpp/ql/lib/semmle/code/cpp/ir/dataflow/internal/DataFlowPrivate.qll

```diff
@@ -907,21 +1006,200 @@ class CastNode extends Node {
   CastNode() { none() } // stub implementation
 }
 
+cached
+newtype TDataFlowCallable =
```

Comment:
I hope we can make this private?
```suggestion
private newtype TDataFlowCallable =
```

Reply:
I *hope* so too, I'll accept the change and then test it locally.